support/modules/rpm-install

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+STATE="$1"
+FILES="$2"
+
+case "$STATE" in
+    ArtifactInstall)
+        rpm -U "$FILES"/files/*.deb
```

Comment:
above should probably be `*.rpm` not `*.deb`

Reply:
Good eye! Fixed locally.